backend/src/main/java/com/example/backend/controller/TripsController.java

```diff
@@ -18,8 +18,15 @@ public List<Trip> getAll() {
         return this.tripService.getAll();
     }
 
+    @GetMapping(path = "/{Id}")
+    public Trip getById (@PathVariable String Id) throws Exception {
```

Comment:
Vielleicht als Anmerkung für zukünftige Sachen: Best practice in Java ist glaub ich, dass variablen alle in PascalCase sind, also klein anfangen, in diesem fall: `id`. 
Is aus meiner Sicht jetzt aber nicht so wichtig, dass du das hier unbedingt ändern muss. 
Würde ich dir überlassen.